src/helpers/git_helpers/merge_helpers.sh

```diff
@@ -33,9 +33,12 @@ function commit_squash_merge {
   local branch_name=$1
   shift
   local options=$(parameters_as_string "$@")
-  local author=$(branch_author "$branch_name")
-  if [ "$author" != "$(local_author)" ]; then
-    options="--author=\"$author\" $options"
+  if ! [[ options == *"--author"* ]]; then
```

Comment:
Got it from [here](http://stackoverflow.com/questions/229551/string-contains-in-bash)